src/azure-cli/azure/cli/command_modules/appservice/static_sites.py

```diff
@@ -120,7 +120,7 @@ def set_staticsite_function_app_settings(cmd, name, setting_pairs, resource_grou
         setting_dict[key] = value
 
     return client.create_or_update_static_site_function_app_settings(
-        resource_group_name, name, kind=None, properties=setting_dict)
```

Comment:
Is `kind` removed?

Reply:
Is it caused by SDK upgrade?

Reply:
ya I think so. the latest SDK here expects parameter of app_settings rather than kind.
https://github.com/Azure/azure-sdk-for-python/blob/cac6069ae45c18810fa912327c41b7dbd452d533/sdk/appservice/azure-mgmt-web/azure/mgmt/web/v2020_09_01/operations/_static_sites_operations.py#L1195